packages/now-next/src/index.ts

```diff
@@ -334,7 +335,19 @@ export const build = async ({
   const memoryToConsume = Math.floor(os.totalmem() / 1024 ** 2) - 128;
   const env: { [key: string]: string | undefined } = { ...spawnOpts.env };
   env.NODE_OPTIONS = `--max_old_space_size=${memoryToConsume}`;
-  await runPackageJsonScript(entryPath, shouldRunScript, { ...spawnOpts, env });
+
+  if (config.buildCommand) {
+    await execCommand(config.buildCommand, {
```

Comment:
```suggestion
    console.log(`Running "${config.buildCommand}"`);
    await execCommand(config.buildCommand, {
```
Should we log this like we do when calling `runPackageJsonScript`?